pages/apply/index.js

```diff
@@ -5,10 +5,6 @@ import Nav from "../../components/nav";
 import { Container, Row, Col } from "reactstrap";
 import Link from "next/link";
 import Footer from "../../components/footer";
-import ReactGA from 'react-ga';
-
-ReactGA.initialize('UA-143288182-1');
-ReactGA.pageview('/apply');
 
 const ApplyPage = () => (
```

Comment:
this is a short hand way to define react components, so you can't define `componentDidMount`. Assuming thats why you couldn't add google analytics, all you have to do is convert this to look like a regular component.
`class ApplyPage extends React.Component {` and then move the jsx code to be returned by a render function.